service/storage_service.cc

```diff
@@ -1466,9 +1466,24 @@ class topology_coordinator {
             auto reason = ::format(
                 "insert CDC generation data (UUID: {})", gen_uuid);
             co_await update_topology_state(std::move(guard), {std::move(mutation), builder.build()}, reason);
+            break;
         }
+        case global_topology_request::keyspace_rf_change: {
+            slogger.info("raft topology: new keyspace RF change requested");
+
+            auto tmptr = get_token_metadata_ptr();
+            auto [ks_name, rf_per_dc, guard_, mutation] = co_await prepare_and_broadcast_keyspace_rf_change_data(tmptr, std::move(guard), std::nullopt);
+            guard = std::move(guard_);
+
+            topology_mutation_builder builder(guard.write_timestamp());
+            builder.set_transition_state(topology::transition_state::tablet_migration)
+                    .set_new_keyspace_rf_change_data(ks_name, rf_per_dc);
+            auto reason = ::format(
+                    "insert keyspace RF change data (ks name: {}, RF per DC: {})", ks_name, rf_per_dc);
+            co_await update_topology_state(std::move(guard), {std::move(mutation), builder.build()}, reason);
```

Comment:
This hunk doesn't belong to this commit (which is about refactoring).

Reply:
Right, sorry, removed.